api-tests/data/role-service/roleservice.js

```diff
@@ -0,0 +1,28 @@
+const validRole = {
+    "id": makeid(7),
+}
+
+const emptyString = {
+    "id": "",
+}
+
+const tooLongId = {
+    "id": makeid(256),
+}
+
+
+function makeid(length) {
```

Comment:
It looks as though the purpose of this function is to generate a random alphanumeric string?    I expect you will want to reuse this in other tests going forward, would it be worth naming it something more generic like "makeRandomString"?   
I don't know much about javascript libraries but I wondered whether there is a library function for generating strings, to save you writing this loop by hand?

Reply:
I have updated the name as suggested but with regards to using a library and avoiding the loop, its not possible in this scenario as I am pulling the length of the string as well for 2 scenarios.